packages/ui/components/document/document-share-button.tsx

```diff
@@ -71,7 +71,7 @@ export const DocumentShareButton = ({ token, documentId, className }: DocumentSh
 
   const onTweetClick = async () => {
     let { slug = '' } = shareLink || {};
-
+    await fetch(`${process.env.NEXT_PUBLIC_WEBAPP_URL}/share/${slug}/opengraph`);
```

Comment:
Awesome! Mind dropping a comment as to why we're doing this so no one deletes it in the future thinking we're being silly?

We may also want this after our `!slug` check to ensure we always have a valid share link to prewarm.

Reply:
Sure, I've just added the changes.